common/azure-common/src/main/java/com/azure/common/exception/HttpRequestException.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.common.exception;
+
+import com.azure.common.http.HttpResponse;
+
+/**
+ * The exception thrown when received unsuccessful response with http status code (e.g. 3XX, 4XX, 5XX) from the service request.
+ *
+ * @see ResourceModifiedException
+ * @see ResourceExistsException
+ * @see ClientAuthenticationException
+ * @see ResourceNotFoundException
+ * @see ServerException
+ * @see TooManyRedirectsException
```

Comment:
Only use `@see` to link to relevant exceptions. Here I would probably only link to `@ServiceRequestException`. For other classes, please ensure the same.